distribution/docs/src/main/resources/content/_securityFramework/federated-identity.adoc

```diff
@@ -0,0 +1,65 @@
+:title: Federated Identity
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:order: 10
+:summary: How a user's identity is shared with federated sources during queries
+
+Identity federation is the sharing of a user's identity and attributes among separate systems. In
+{branding}, it is used when performing federated queries - the user's identity is attached to outgoing
+requests, allowing federated sources to run queries as that user. This is critical because {branding}
+must make authorization decisions for all queries, not just those initiated by the local node.
+
+To represent a user's identity when federating, {branding} uses a SAML assertion. This is the format
```

Comment:
It can be helpful to readers when documentation has few pronouns. 
Instead of "This is the format..", repeat the referent:

`SAML assertions are the format...`

Also, the sentence reads well without the phrase "are the format". 

`SAML assertions are used...`